### PR TITLE
VPN landing page copy tweaks (#9950)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
http://localhost:8000/en-US/products/vpn/

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/9950

## Testing
- [ ] Landing page copy for number of servers should reflect the variable change.
- [ ] FAQ list of compatible devices should use bullets instead of numbers.

### CODE REVIEW

**Comment by alexgibson** [comment]
Added a couple of last minute copy tweaks, so bumping this to a P1 as it blocks getting things into Pontoon.

**Review by pmac — APPROVED** [review]
🍰

**Comment by alexgibson** [comment]
Circle Python tests seem to be stuck in a pending state here, but clicking the details link it looks like everything passed. There's a message saying "GitHub returned status 500 adding commit status of success", so I think we're fine to merge here.

File: bedrock/products/templates/products/vpn/landing.html [hunk 1]
```diff
@@ -192,7 +192,7 @@ <h3>{{ ftl('vpn-landing-faq-compatibility-question-heading') }}</h3>
       </summary>
       <p class="vpn-faq-item-detail">{{ ftl('vpn-landing-faq-compatibility-question-desc') }}</p>
 
-      <ol class="vpn-faq-item-detail mzp-u-list-styled">
+      <ul class="vpn-faq-item-detail mzp-u-list-styled">
```
Comment (craigcook) [inline bedrock/products/templates/products/vpn/landing.html:195]: Need to change the closing tag to `</ul>` as well.

Comment (pmac) [reply bedrock/products/templates/products/vpn/landing.html:195]: AH! Reviewer fail. Thanks Craig.

Comment (craigcook) [reply bedrock/products/templates/products/vpn/landing.html:195]: D'oh, I commented just as it was merged. This shouldn't cause any visible issues, most browsers will be able to handle it. Just a minor fix we can do for the next time the page changes.

Comment (craigcook) [reply bedrock/products/templates/products/vpn/landing.html:195]: Fixed in #10029

**Pull request merged**